Sources/WWDCNotes/WWDCNotes.docc/WWDC24/WWDC24-10136-Whats-new-in-Swift.md

```diff
@@ -8,11 +8,247 @@ Join us for an update on Swift. We’ll briefly go through a history of Swift ov
    @CallToAction(url: "https://developer.apple.com/wwdc24/10136", purpose: link, label: "Watch Video (30 min)")
 
    @Contributors {
-      @GitHubUser(<replace this with your GitHub handle>)
+      @GitHubUser("MortenGregersen")
```

Comment:
Please don't add quotes to your handle. I've just updated the Contributing document to make it more clear, it was ambiguous before.

Reply:
I will remove them. You are welcome to do small fixes like this yourself 😉